### ISSUE TITLE
[Bug]: Cannot install due to lack of space, even though there is almost 400 GB of space

### ISSUE THREAD

**Issue by DraginEye** [Comfy-Org/desktop#584]
### App Version

ComfyUI Setup 0.4.5 - x64.exe

### Expected Behavior

Install normally

### Actual Behavior

![image](https://github.com/user-attachments/assets/77049d11-99a6-46d4-8faa-56c88961bc7a)
![image](https://github.com/user-attachments/assets/e890a0e4-ec6a-468b-bca4-b3d58e6c8ade)
![image](https://github.com/user-attachments/assets/e97c1189-c3a5-4770-b389-c696032678fa)

### Steps to Reproduce

Try to install, bypass inaccurate warning about not having an nvidia gpu.

### Debug Logs

```powershell
unavailable
```

### Browser Logs

unavailable

### Setting JSON

unavailable

**Comment by robinjhuang** [comment]
Thanks for flagging this. Were you able to bypass the warnings?

**Comment by DraginEye** [comment]
i can bypass the warning for the graphics card but i am unable to bypass the no space warning

**Comment by DraginEye** [comment]
DeviceID     : C:
DriveType    : 3
ProviderName :
FreeSpace    : 393722658816
Size         : 2047760920576
VolumeName   : Windows 10

DeviceID     : D:
DriveType    : 3
ProviderName :
FreeSpace    : 423756709888
Size         : 2048390066176
VolumeName   : Drive

DeviceID     : E:
DriveType    : 3
ProviderName :
FreeSpace    : 428988084224
Size         : 2000381014016
VolumeName   : Drive


i dont know how to format it into a code block but here it is

**Comment by DraginEye** [comment]
'powershell.exe' is not recognized as an internal or external command,
operable program or batch file.

**Comment by DraginEye** [comment]
![image](https://github.com/user-attachments/assets/d88af885-4e19-4b31-ae56-994517a6c855)

**Comment by DraginEye** [comment]
```
Name                           Value
----                           -----
PSVersion                      5.1.19041.5247
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.5247
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by DraginEye** [comment]
yes, and it doesnt say i have the wrong GPU anymore either it brings up the get started splash screen.

**Comment by marioja** [comment]
I get the same error with no way to bypass the disk space problem on windows 11 24H2 and here is what powershell 5.1 installed returns:

```
(get-command powershell).definition
C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
```
I am running latest version 0.4.72 and here is the screen shot:

<img width="817" height="745" alt="Image" src="https://github.com/user-attachments/assets/026fbcec-c8aa-432b-9460-783c32264004" />

There is 1TB of free disk space.  I tried other disks with the same error.

```
Get-WmiObject Win32_logicaldisk


DeviceID     : C:
DriveType    : 3
ProviderName :
FreeSpace    : 1200822419456
Size         : 3999623278592
VolumeName   :
```

Can you reopen this issue?

**Comment by marioja** [comment]
I have powershell 7.5.3 installed also.  i don't know if that is a factor.
